src/components/SelectionToolbar.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { Portal } from "react-portal";
+import { EditorView } from "prosemirror-view";
+import getTableColMenuItems from "../menus/tableCol";
+import getTableRowMenuItems from "../menus/tableRow";
+import getTableMenuItems from "../menus/table";
+import getFormattingMenuItems from "../menus/formatting";
+import FloatingToolbar from "./FloatingToolbar";
```

Comment:
Previously, this file was the business logic from `FloatingToolbar`. Now we're moving the business logic into `SelectionToolbar` and letting `FloatingToolbar` handle the display logic